client/src/pages/CoursesPage/useCourses.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState, useMemo } from "react";
+import MomentumClient from "../../common/MomentumClient";
+
+const courseTableColumns = [
+    {
+        Header: "Course Code",
+        accessor: "courseCode",
+    },
+    {
+        Header: "Course Name",
+        accessor: "name",
+    },
+    {
+        Header: "Semester",
+        accessor: "semester",
+    },
+    {
+        Header: "Points",
+        accessor: "points",
+    },
+];
+
+const useCourses = () => {
+    const [courses, setCourses] = useState([]);
+
+    const fetchAllCourses = () => {
+        MomentumClient.getCourses().then((res) => {
+            setCourses(res);
+        }).catch(e => {
+            console.log(e);
+        });
+    }
+
+    const updateCourse = (course) => {
+        MomentumClient.updateCourse(course).then(r => {
+            fetchAllCourses();
+        }).catch(e => {
+            console.log(e);
+        })
+    }
+
```

Comment:
if it is one line of code, don't add curly braces pls : )

Reply:
Very true sir, thank you